index.js

```diff
@@ -517,7 +517,7 @@ Tree.prototype.previousSibling = function (obj) {
  * If an optional index is received, the node will be inserted at that index within the
  * `to`'s children.
  */
-Tree.prototype.move = function (node, to, idx) {
+Tree.prototype.move = function (node, to, idx, expandTo = true) {
```

Comment:
To me the name `expandTo` would suggest you could supply a specific ID to expand to.  What would you think of calling it `expand` or `expandAncestors` instead?

Reply:
Good idea. I'll change it.